src/index.js

```diff
@@ -132,14 +183,16 @@ function rejectTrip(tripId) {
   console.log('Rejected')
 }
 
-function refreshData() {
+function refreshUserData(userData) {
   Promise.all([
     getData('travelers', travelers),
     getData('trips', trips),
     getData('destinations', destinations)
   ])
-  .then(data => {
-    convertDataIntoClassInstances()
-    displayTrips()
+    .then(() => {
+      user = travelers.find(traveler => traveler.id === userData.id)
+      convertDataIntoClassInstances()
+      displayUserDashboard(user)
+      console.log('WE REFRESH THIS', user.trips)
```

Comment:
If you wanted user experience, you could try replacing the console.log with an alert. Don't know how well that'll go over with it being called in a promise, but it's an idea.